### PR TITLE
Added short generator & preserve image option

### PR DESCRIPTION
**Pull request by Bladrak**

### CODE REVIEW

File: tc_shortener/generators/__init__.py [hunk 1]
```diff
@@ -4,8 +4,34 @@
...
+        if self.context.config.get('SHORTENER_GENERATOR_PRESERVE_NAME'):
+            reg = re.compile(Url.regex())
+            result = reg.match(url)
+
+            if result == None:
+                raise ValueError("URL does not match thumbor's URL pattern")
+
+            result = result.groupdict()
+
+            image = "/{image}".format(image = result['image'])
```
Comment (masom) [inline tc_shortener/generators/__init__.py]: PEP8!

**Comment by masom** [comment]
:+1:

**Comment by Bladrak** [comment]
Fixed @masom ;)

**Comment by masom** [comment]
Great! Using the shortuuid instead of sha256 is a better default.

**Pull request merged**